app/models/mixins/service_mixin.rb

```diff
@@ -18,29 +18,33 @@ def add_resource(rsc, options = {})
     rsc_type = rsc.class.base_class.name.tableize
     raise _("Cannot connect service with nil ID.") if rsc.id.nil? && rsc_type == "service_templates"
 
+    # what is this?
```

Comment:
These comments aren't very helpful for future readers. What about this do you not understand? Can you give more detail or at least ping someone for info?